scripts/main.js

```diff
@@ -71,4 +72,68 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log("De afdeling sales heeft", departments.sales.numberOfEmployees, "medewerkers.");
+
+console.log("Marketing is een leuke afdeling om te werken.", departments.marketing.description)
+
+console.log("De afdeling Customer Service heeft", departments["customer-service"].numberOfEmployees, "medewerkers")
+
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager.", departments.sales.jobs[1].description)
+
+const userName = prompt('Hoi! Hoe heet je?');
+console.log(userName);
+//
+// document.getElementById('name').textContent = userName;
+
+const userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+console.log(userInput);
+
+
+
+// switch (userInput) {
+//     case "sales":
+//     case "marketing":
+//     case "customer-service":
+//         console.log("Je koos", userInput, ".", departments[userInput].description)
+//         break
+// }
+
+if ((userInput === "marketing") || (userInput === "sales") || (userInput === "customer-service")) {
+    console.log("Je koos", userInput, ".", departments[userInput].description)
+} else {
+    console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen. Let erop dat je geen hoofdletters gebruikt in jou antwoord.")
+}
+
+console.log(`${userInput} is een leuke afdeling om te werken. Er werken op dit moment ${departments[userInput].numberOfEmployees} medewerker` )
+
```

Comment:
Hier mis ik bij markting de job "Marktonderzoekanalist".

De kunst is hier om de opties 0 tm 4 variable te maken. Is de input sales, dan moeten  er 4 opties getoond worden.
Is de input Marketing dan moeten er 5 opties getoond worden.

const jobs = departments[job].jobs;
let jobString = '';
for(let i=0;i <jobs.length;i++){
    jobString += i+" "+jobs[i].title+"\n";
}

userInput = prompt(`Je koos voor ${job}. Over welke functie wil je meer weten? Voer een getal tussen 0 en ${(jobs.length -1)} in. \n\n ${jobString}`);